iam/api-client/requirements.txt

```diff
@@ -1,5 +1,5 @@
 google-api-python-client==2.25.0
 google-auth==2.0.0
 google-auth-httplib2==0.1.0
-boto3==1.19.4
-botocore==1.22.6
+boto3==1.20.2
+botocore==1.23.2
```

Comment:
This needs to happen together, otherwise pip raises an error:

```
The conflict is caused by:
    The user requested botocore==1.22.6
    boto3 1.20.2 depends on botocore<1.24.0 and >=1.23.2
```